### DISCUSSION TITLE
http redirect

### DISCUSSION THREAD

**Discussion by darrenwoods3** [netbox-community/netbox#9849] [Q&A] [answered]
I've installed netbox running apache.  Is there a way to do a redirect/rewrite from http to https?

**Comment by candlerb** [comment, marked as answer]
Yes - although "how do I configure Apache?" isn't really a Netbox question.

The following should be all you need:

```
<VirtualHost *:80>
    Redirect permanent / https://netbox.my-domain.com/
</VirtualHost>

<VirtualHost *:443>
    ... standard Netbox config goes here
</VirtualHost>
```

If you have an existing "default" site, then you may need to disable it (e.g. for Ubuntu: `a2dissite 000-default`)
